src/konnektor/network_planners/generators/explicit_network_generator.py

```diff
@@ -44,24 +44,30 @@ def __init__(
     def generate_ligand_network(
         self,
         edges: Iterable[tuple[Component, Component]],
+        nodes: Iterable[Component] | None = None,
     ) -> LigandNetwork:
         """
         Create a network with pre-defined edges.
+        The network can be defined by specifying only edges, in which case the nodes are implicitly added.
 
-        This can be used as initial_edge_lister
+        This can be used as initial_edge_lister.
```

Comment:
Not for this PR, but `initial_edge_lister` just doesn't make a lot of sense to a user casually looking to use this method.

Reply:
I agree and updated it here since it's a quick fix (I should still do a docs-specific PR soon)